docs/docs/tracking.md

```diff
@@ -0,0 +1,73 @@
+Pathfora can send tracking events and the user data submitted to Lytics and Google Analytics, as long as your tags are all set up in the correct order, module data will be sent automatically. Below we will look at which fields are sent in detail and their formats.
+
+## Lytics
+As long as your [Lytics javascript tag](#) is loaded before the Pathfora tag, all event data, and data collected from modules with user input will be sent to the `default` stream in your Lytics account. The following raw data fields can be sent to Lytics by Pathfora.
```

Comment:
Whoops this is wrong, it uses `jstag.send` so it will use whatever stream is configured there.